src/domUpdates.js

```diff
@@ -0,0 +1,146 @@
+const recipeCards = document.querySelector('.all-cards');
+const favButton = document.querySelector('.view-favorites');
+
+
+let domUpdates = {
```

Comment:
Awesome job getting all of this moved out of scripts and debugging recipeRepo coming across the files. This is a really good chunk of DOM code that is ready for any team member to jump on and debug without creating conflicts, so thank you for that